salt/fileserver/__init__.py

```diff
@@ -533,15 +523,6 @@ def serve_file(self, load):
         '''
         ret = {'data': '',
                'dest': ''}
-        if 'env' in load:
-            salt.utils.warn_until(
-                'Boron',
-                'Passing a salt environment should be done using \'saltenv\' '
-                'not \'env\'. This functionality will be removed in Salt '
-                'Boron.'
-            )
-            load['saltenv'] = load.pop('env')
```

Comment:
For all of these, we probably still want to check if `env` is in the load and delete it if so (while warning).